tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests"""
```

Comment:
## Little bit of general setup required
There are 2 checks that came out of my sync subclass PR that I wanted to preserve. As such, this entails ideally multiple files, which further entails some setup. An `__init__.py` and a `config.py` for allowing imports for DRYness, etc.